src/app/lib/utilities/ichefURL/index.js

```diff
@@ -38,13 +39,10 @@ const buildIChefURL = ({ originCode, locator, resolution, isWebP = false }) => {
     .filter(Boolean)
     .join('/');
 
-  const isWebPSupported = isWebP && WEBP_ORIGIN_CODES.includes(originCode);
-
-  if (isWebPSupported) {
-    return `${url}.webp`;
+  if (url.endsWith('.webp')) {
+    return url;
   }
```

Comment:
Consider this to shorten the return section of the function. This approach applies here 
since it's not a [guard clause](https://github.com/bbc/simorgh/pull/11575/files#diff-52550de5c2974525a080b5738d3ac102090a1c226f660df5183015286b2129bbR27-R29) and there are two options for which we can leverage the ternary operator for.


With this, ln45 -> [`return `${url}.webp`;`](https://github.com/bbc/simorgh/pull/11575/files#diff-52550de5c2974525a080b5738d3ac102090a1c226f660df5183015286b2129bbR45), shouldn't be needed.
```suggestion
  return url.endsWith('.webp') ? url : `${url}.webp`;
```

Reply:
Hows do the change in this file relate to the suggested BFF change to
ensure all image URLs are set to `.webp` by the BFF?
Should we use the BFF to transform image URLs long term?
See [comment](https://jira.dev.bbc.co.uk/browse/WSTEAMA-749?focusedId=6260699&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6260699) for reference.